src/providers/postgres/qgspostgresprovider.cpp

```diff
@@ -2212,7 +2227,11 @@ bool QgsPostgresProvider::parseDomainCheckConstraint( QStringList &enumValues, c
       if ( arrayPosition != -1 )
       {
         QString valueList = checkDefinition.mid( arrayPosition + 6, closingBracketPos );
+#if QT_VERSION < QT_VERSION_CHECK(5, 15, 0)
+        const QStringList commaSeparation = valueList.split( ',', QString::SkipEmptyParts );
+#else
```

Comment:
this can be dropped, we require 5.15 now

Reply:
indeed